android/app/src/main/res/values-es/strings.xml

```diff
@@ -634,6 +618,8 @@
     <string name="bookmark_lists_hide_all">Ocultar todo</string>
     <string name="bookmark_lists_show_all">Mostrar todo</string>
     <plurals name="bookmarks_places">
+        <item quantity="one"/>
+        <item quantity="many"/>
```

Comment:
This may have a logical difference in practice, worth testing to ensure one/many/other bookmark quantities still show as intended

Reply:
https://github.com/organicmaps/organicmaps/pull/10321